### DISCUSSION TITLE
Widget requirements

### DISCUSSION THREAD

**Discussion by LemonDrug** [DustinBrett/daedalOS#424] [Feature Requests]
Hello friend, first of all I want to say that this project of yours is really great, there is nothing better than this.
But there is currently a problem. Chinese law requires that the registration number must be displayed on the website, and this string of numbers must have a hyperlink. I really don't have the ability to add a small component like a red box at the bottom of your project desktop and fill in the registration number, so I can only raise my need here and hope to get support. I am very happy.

<img width="1440" alt="image" src="https://github.com/DustinBrett/daedalOS/assets/20113317/e359e052-96ee-47df-98c9-921859fdde38">

**Comment by LemonDrug** [comment]
If it is not convenient to make widgets, maybe there are two boxes in the lower right corner like time, where you can fill in text, and when clicked, it will automatically jump to other websites. This is also good.
If you have a better way, that's great too, thanks again.
<img width="1440" alt="image" src="https://github.com/DustinBrett/daedalOS/assets/20113317/16651d50-9f22-44c0-a371-d8faa976abd9">

**Comment by DustinBrett** [comment]
Thanks for checking out my site! The idea of widgets is interesting and I do have a long term plan to allow 3rd party plugins/widgets which would make it easy to do what you want. If you plan to host your own instance of daedalOS then you can modify the code to achieve this, but you will need some basic React knowledge. You could even hack something together with a `<script>` tag and pure JavaScript which would place a link on the page that is outside the normal flow of the app.
